DocumentsFromSnapshotMigration/src/test/java/com/rfs/ParallelDocumentMigrationsTest.java

```diff
@@ -52,6 +52,7 @@ public static Stream<Arguments> makeDocumentMigrationArgs() {
         var targetImageNames = TARGET_IMAGES.stream()
             .collect(Collectors.toList());
         var numWorkersList = List.of(1, 3, 40);
+        var compressionEnabledList = List.of(true, false);
```

Comment:
I don't think that you're splicing these values into the return value.

Reply:
Fixed